mlx/traceability.py

```diff
@@ -249,6 +253,38 @@ def run(self):
 
         return [item_list_node]
 
+class ItemLinkDirective(Directive):
+    """
+    Directive to generate a list of items.
+
+    Syntax::
+
+      .. item-link::
+         :sources: list_of_items
+         :targets: list_of_items
+         :type: relationship_type
+
+    """
+    final_argument_whitespace = True
+    # Options
+    option_spec = {'sources': directives.unchanged,
+                   'targets': directives.unchanged,
+                   'type': directives.unchanged}
+    # Content disallowed
+    has_content = False
+
+    def run(self):
+        env = self.state.document.settings.env
+        app = env.app
+
+        item_link_node = ItemLink('')
+
+        item_link_node['sources'] = self.options['sources']
```

Comment:
`sources` is a string, you need to `split()` on spaces to get the different items (see line 477 of same file for example)

same for `targets`